
geometry_msgs/PoseArray  positions
std_msgs/Int32MultiArray ids
std_msgs/Int32MultiArray stay_times
bool                     running_status
std_msgs/Int32MultiArray particular_station_indexs
